presto-docs/src/main/sphinx/connector/cassandra.rst

```diff
@@ -211,6 +211,7 @@ TIMEUUID          VARCHAR
 TINYINT           TINYINT
 VARCHAR           VARCHAR
 VARIANT           VARCHAR
+UDT               VARCHAR
```

Comment:
Why do you convert `UDT` to `VARCHAR`. Isn't `JSON` (or `map`) a better representation?

Reply:
As we talked offline, I'll map to presto `row` or `json`.